addons/mail/static/src/components/thread_view_topbar/thread_view_topbar.js

```diff
@@ -4,7 +4,35 @@ import { registerMessagingComponent } from '@mail/utils/messaging_component';
 import { useRefToModel } from '@mail/component_hooks/use_ref_to_model/use_ref_to_model';
 import { useUpdateToModel } from '@mail/component_hooks/use_update_to_model/use_update_to_model';
 
+import { usePosition } from '@web/core/position/position_hook';
+
 const { Component } = owl;
+const { useRef } = owl.hooks;
+const { xml } = owl.tags;
+
+class ThreadViewTopbarInvitePopover extends Component {
+
+    setup() {
+        super.setup();
+        usePosition(this.props.target, {
+            margin: 16,
+            position: this.props.position,
+        });
+    }
+
+}
```

Comment:
make generic, with t-component, t-props

Reply:
and make a popover model, the generic component should take no other prop than its localId